### ISSUE TITLE
Persist the current tab selection in the menu

### ISSUE THREAD

**Issue by christianpoveda** [austinjones/tab-rs#318]
Hi

I was wondering if it makes sense to persist the selection of the current tab in the menu. What I mean is that if you have three tabs:
```
tab1
tab2
tab3
```
And you are in `tab2` then when you open the menu using `Ctrl+T` then you should see 
```
> tab2
tab1
tab3
```
then if you switch to `tab1` you should see
```
> tab1
tab2
tab3
```
In that way the selected tab is always first and when changing tab you instinctively know you have to go down to select a different tab instead of remembering the name of the current tab and move somewhere else. This is mostly a "muscular memory" thing for me but I think it is a sensible behavior.

I don't want to impose my workflow as the "right thing" so I think this could be added as a configurable behavior. And I propose myself to implement this if this gets a green light.

PD: I tried to check if this is already implemented of if this is a duplicated issue but did not found anything in the issue tracker and documentation.

PD: An alternative would be to keep the order of the tabs in the menu but keep the current tab as the selected when entering. So if you are in `tab2` and open the menu, you'd see:
```
tab1
> tab2
tab3
```

**Comment by austinjones** [comment]
Hi @christianpoveda,
I'm planning two related changes which are scoped for v0.6.0.  I think this should be the default behavior - I run into it all the time and several people have suggested it.  The related issues are:

1. The list of tabs in the fuzzy finder will be sorted by 'most recently selected' if you haven't entered a query.  That way anything you've used recently will be at the top.  I'm considering switching the top two entries, so `ctrl-T enter` would work like alt-tab, switching between sessions.  See #265 
2. The `ctrl-T ESC` behavior will be changed so that it will return to the most recently selected tab (unless you just opened the fuzzy finder from your root shell).  I think this will be a nice improvement so you can 'cancel' a fuzzy finder session if you realize you don't want to switch.  See #314.

I have some of this implemented in PR #298, but it can't be merged until the final v0.5.X patch release (which I'm hoping to push this week).  Once I've merged that PR, the functionality in #314 should be straightforward if you'd like to take that on.

**Comment by christianpoveda** [comment]
Oh that's great! I'm closing this as it will be redundant when those PR are merged :) Thank you!
